daprdocs/content/en/concepts/observability-concept.md

```diff
@@ -4,60 +4,38 @@ title: "Observability"
 linkTitle: "Observability"
 weight: 500
 description: >
-  How to monitor applications through tracing, metrics, logs and health
+  Monitor applications through tracing, metrics, logs and health
 ---
 
-Observability is a term from control theory. Observability means you can answer questions about what's happening on the inside of a system by observing the outside of the system, without having to ship new code to answer new questions. Observability is critical in production environments and services to debug, operate and monitor Dapr system services, components and user applications.
+When building an applications, understanding how the system is behaving is an important part of operating it - this includes having the ability to observe the internal flows of an application, gauging its performance and becoming aware of problems as soon as they occur. This is challenging for any system but even more so for a distributed system comprised of multiple microservices where a flow may start in one microservices but continue in another. Observability is critical in production environments but also useful during development to understand bottlenecks, improve performance and perform basic debugging across the span of microservices.
```

Comment:
you use the word "floww" here in this paragraph. "Calls" is usually a more common and typical term.

Reply:
changes where it made sense and clarified where it is meant to mean a series of calls